draft-ietf-idr-deprecate-as-set-confed-set.xml

```diff
@@ -105,27 +105,34 @@ title) for use on http://www.rfc-editor.org/rfcsearch.html.
 
     <abstract>
       <t>BCP 172 (i.e., RFC 6472) recommends not using AS_SET and AS_CONFED_SET
-      in the Border Gateway Protocol. This document advances this recommendation
-      to a standards requirement in BGP; it proscribes the use of the AS_SET
-      type of path segments in the AS_PATH.  This is done to simplify the design
-      and implementation of BGP and to make the semantics of the originator of a
-      route clearer. This will also simplify the design, implementation, and
-      deployment of various BGP security mechanisms. This document (if approved)
-      updates RFC 4271 and RFC 5065, and obsoletes RFC 6472.</t> </abstract>
-      </front>
+      AS_PATH segment types in the Border Gateway Protocol (BGP). This document
+      advances that recommendation to a standards requirement in BGP; it
+      proscribes the use of the AS_SET and AS_CONFED_SET path segment typess
+      in the AS_PATH.  This is done to simplify the design and implementation
+      of BGP and to make the semantics of the originator of a BGP route clearer.
+      This will also simplify the design, implementation, and deployment of
+      various BGP security mechanisms. This document, if approved, updates RFC

```

Comment:
I'd suggest instead: 
"This document updates RFC 4271 and RFC 5065, and obsoletes RFC 6472." and then add "(if approved)" to the Updates: header.

The reason for this is that it would be embarrassing if we forgot this text in the final RFC version.

Reply:
Whoops, apologies, I hadn't checked the rendered output and had forgotten that XML2RFC / etc automagically add "(if approved)" (e.g: "Updates: 4271 5065 (if approved)").

So, we are just waiting on Sriram to submit I think...